### DISCUSSION TITLE
Does nicegui support third-party vue dependencies now?

### DISCUSSION THREAD

**Discussion by Autism-al** [zauberzeug/nicegui#3790] [Q&A]
I would like to use some dependencies such as element-plus，but I only found cases of js dependencies in examples

**Comment by falkoschindler** [comment]
Hi @Autism-al,

Unfortunately that's not the case. As I noticed while working on PR #2991:

> No, there's currently no point in trying to integrate third-party Vue components since we're not able to support Vue 3 components at the moment.

Discussions #1198 and #1454 are also related. The main problem is that NiceGUI uses [vbuild](https://github.com/manatlan/vbuild) for compiling *.vue to *.js, which doesn't support Vue 3 and isn't maintained anymore. We either need to find another compiler, or we also remove support for Vue 2 components like plotly.vue and joystick.vue. (Vue 2 has reached end of life in December 2023.)
